pkg/auth/authenticator/basic/basic.go

```diff
@@ -0,0 +1,48 @@
+package basic
```

Comment:
There is already a basicauth authenticator in plugin/pkg/auth/authenticator/request/basicauth